src/08_comprehensions.py

```diff
@@ -26,16 +26,16 @@
 
 a = ["foo", "bar", "baz"]
 
-y = []
+y = [y.upper() for y in a]
 
 print(y)
 
 # Use a list comprehension to create a list containing only the _even_ elements
 # the user entered into list x.
 
-x = input("Enter comma-separated numbers: ").split(',')
+x = [1,5,6,4,9,4,6,3,1,25,4,6]
```

Comment:
It is looking for an input here. `x = input("Enter comma-separated numbers: ").split(',')` Maybe you switched that out for testing?